package.json

```diff
@@ -42,11 +42,11 @@
     "lint:prettier": "prettier --write --loglevel warn",
     "lint:typecheck": "nuxt typecheck",
     "prepack": "pnpm build",
-    "prepare": "husky install",
     "prepublishOnly": "pnpm lint && pnpm test && pinst --disable",
     "release": "bumpp && npm publish",
     "test": "vitest run",
     "test:types": "tsc --noEmit",
+    "postinstall": "husky install",
     "postpublish": "pinst --enable"
```

Comment:
Is `pinst` required because this module can be required by projects that use `yarn`? I'm wondering because the `pinst` README mentions it only being useful for usage with `yarn` and this module uses `pnpm`.

Reply:
oh, nice find! we can move it back to `prepare` and remove `pinst` entirely then